lms/djangoapps/certificates/views/webview.py

```diff
@@ -97,7 +97,7 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
-        month=user_certificate.modified_date.strftime("%B"),
+        month=_(user_certificate.modified_date.strftime("%B")),
```

Comment:
Now I understand what you mean.
@pepeportela Could you develop a solution as @amangano-edx says, using a helper method or an alternative solution?
@amangano-edx Is it better to create a new pull request and close this referencing the new PR?